README.md

```diff
@@ -16,10 +15,8 @@ data.
 
 ## Installation
 
-``` r
-# install.packages("remotes")
-remotes::install_github("poissonconsulting/poisspatial")
-```
+    # install.packages("remotes")
```

Comment:
I'm wondering if our installations are doing something different because when I pull the code and knit the README.Rmd into the README.md file I get

```
## Installation

``` r
# install.packages("remotes")
remotes::install_github("poissonconsulting/poisspatial")
`` `
```

Note I had to add the space to `` ` to make it view correctly in this issue.

Can you confirm what you get when you knit the README.md file?

Reply:
@sarahLy9 - can you resolve this before merging?